planning/backlog.md

```diff
@@ -8,42 +8,35 @@ A collection of user stories you can choose from. Stories inside of each priorit
 
 are necessary for basic usability:
 
-- **user story name**: As a [type of user] I want to [do something] so that [I achieve some goal]
-  - Given [context] when [a specific action is performed] then [a set of consequences should occur]
-  - ...
-- **user story name**: As a [type of user] I want to [do something] so that [I achieve some goal]
-  - Given [context] when [a specific action is performed] then [a set of consequences should occur]
-  - ...
-- ...
+- **Website Visitor**: As a Website Visitor I want to have a clear navigation around the website   
+  - [ ] Given when I see the header and the footer, then links to the other parts of the website are provided
+- **Website Visitor**: As a Website Visitor I want to know whether I can use the solution on my computer 
+  - [ ] Given when I visit the website then I can see the list of languages supported, and the technical requirements
+- **Website Visitor**: As a Website Visitor I want to know how to install Loruki on my computer so that I could use it   
+  - [ ] Given when I visit the website then I can see the instruction on installing Loruki  
+- **Website Visitor**: As a User I want to request a demo so that I receive a demo version to my mailbox
+  - [ ] Given when I fill in the form with my contacts then the demo is dent to my e-mail
+- **Admin**: As an Admin I want to send the demo version to the users who requested it so that    
+  - [ ] Given when a website visitor fills in the form, the demo version is automatically sent to their email
+- **Admin**: As an Admin I want to see the collected data of the users who requested the demo version and include them into Loruki mailing list  
+  - [ ] Given when a website visitor fills in the form, I get their email saved into the database
 
 ## Should-Haves
 
 will complete the user experience, but are not necessary:
 
-- **user story name**: As a [type of user] I want to [do something] so that [I achieve some goal]
-  - Given [context] when [a specific action is performed] then [a set of consequences should occur]
-  - ...
-- **user story name**: As a [type of user] I want to [do something] so that [I achieve some goal]
-  - Given [context] when [a specific action is performed] then [a set of consequences should occur]
-  - ...
-- ...
+- **Mobile User**: As a Mobile User I want to easily use the Loruki website from my mobile phone 
+  - [ ] Given when I visit the site from my mobile phone, the web design is responsive
+- **Website Visitor**: As a Website Visitor I want to learn the main statistics of the Loruki solution use so that I know more about its popularity 
+  - [ ] Given when I visit the website I see the information about the number of deployment, projects etc. 
+- **Website Visitor**: As a Website Visitor I want to see Loruki accounts in social media so that I get more information about Loruki 
+  - [ ] Given when go to the header/footer of the page, I see clickable logos of social media that bring me to Loruki's accounts
```

Comment:
Given the website visitor wants to staying in touch with the company, then it should have a contact section in the footer with clickable logos of social media that bring him to Loruki's accounts.